roles/core/time/templates/chrony.conf.j2

```diff
@@ -3,22 +3,18 @@
 ## {{ansible_managed}}
 
 # External servers/pool if asked
-{% if external_time.time_server.server and time_profile == 'server' %}
+{% if time_profile == 'server' %}
```

Comment:
chrony configuration is either server or pool. Using both is redundant. (https://chrony.tuxfamily.org/faq.html#_what_is_the_minimum_recommended_configuration_for_an_ntp_client).
It looks like a bug because we defined both in the examples inventories. Can you revert the change on this file to keep current behaviour? I will open an issue ticket.

Reply:
#166